src/item/StringToItemParser.php

```diff
@@ -1354,6 +1357,7 @@ private static function registerItems(self $result) : void{
 		$result->register("long_weakness_potion", fn() => Items::POTION()->setType(PotionType::LONG_WEAKNESS()));
 		$result->register("long_weakness_splash_potion", fn() => Items::SPLASH_POTION()->setType(PotionType::LONG_WEAKNESS()));
 		$result->register("magma_cream", fn() => Items::MAGMA_CREAM());
+		$result->register("medicine", fn() => Items::MEDICINE());
```

Comment:
I think this can be removed. It's not obvious what type of medicine is removed, and there's no such alias in vanilla anyway.